### PR TITLE
Feature/812 user creates passphrase

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: src/containers/CreatePassphraseScreenContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import { TH1, TP } from '../components/atoms/THeadings';
+import settings from '../settings';
+import theme, { commonStyles } from '../utils/theme';
+import TInfoBox from '../components/TInfoBox';
+import LayoutComponent from '../components/layout';
+import { Props } from '../screens/CreatePassphraseScreen';
+import PassphraseBox from '../components/PassphraseBox';
+
+export default function CreatePassphraseScreenContainer({ navigation }: { navigation: Props['navigation'] }) {
+    const [phraseList, setPhraseList] = useState<string[]>(['barn', 'universe', 'plate', 'star', 'pretty', 'gold']);
```
Comment (theblockstalk) [inline src/containers/CreatePassphraseScreenContainer.tsx:13]: this should be empty by default. but prefilled if not production like we do here
https://github.com/Tonomy-Foundation/Tonomy-ID/blob/development/src/containers/CreateAccountPasswordContainer.tsx#L21

Make sure these 6 world are part of the phrase list

Comment (sadiabbasi) [reply src/containers/CreatePassphraseScreenContainer.tsx]: these boxes are auto filled when clicked on generate right? it's not like writeable?

**Review by theblockstalk — APPROVED** [review]

File: src/containers/CreatePassphraseScreenContainer.tsx (hunk 1 above)
Comment (theblockstalk) [reply src/containers/CreatePassphraseScreenContainer.tsx]: correct. the text boxes are not editable

File: src/containers/CreatePassphraseScreenContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+export default function CreatePassphraseScreenContainer({ navigation }: { navigation: Props['navigation'] }) {
+    const [phraseList, setPhraseList] = useState<string[]>(
+        !settings.isProduction() ? ['barn', 'universe', 'plate', 'star', 'pretty', 'gold'] : ['', '', '', '', '', '']
+    );
+
+    async function onNext() {}
+
+    async function regenerate() {
+        setPhraseList(['gold', 'barn', 'star', 'moon', 'sun', 'tree']);
```
Comment (theblockstalk) [inline src/containers/CreatePassphraseScreenContainer.tsx:20]: this should call `user.generateRandomPassphrase()`

File: src/containers/CreatePassphraseScreenContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+import settings from '../settings';
+import theme, { commonStyles } from '../utils/theme';
+import TInfoBox from '../components/TInfoBox';
+import LayoutComponent from '../components/layout';
+import { Props } from '../screens/CreatePassphraseScreen';
+import PassphraseBox from '../components/PassphraseBox';
+
+export default function CreatePassphraseScreenContainer({ navigation }: { navigation: Props['navigation'] }) {
+    const [phraseList, setPhraseList] = useState<string[]>(
+        !settings.isProduction() ? ['barn', 'universe', 'plate', 'star', 'pretty', 'gold'] : ['', '', '', '', '', '']
```
Comment (theblockstalk) [inline src/containers/CreatePassphraseScreenContainer.tsx:14]: when `isProduction` then the empty strings should be generated `user.generateRandomPassphrase()`

**Pull request merged**